pkg/util/system/util_test_tool.go

```diff
@@ -48,6 +48,10 @@ func NewFileTestUtil(t *testing.T) *FileTestUtil {
 	return &FileTestUtil{TempDir: tempDir, t: t}
 }
 
+func (c *FileTestUtil) Cleanup() {
+	os.RemoveAll(c.TempDir)
```

Comment:
Do we still need the `Cleanup()`? Directories created by `t.TempDir()` are cleaned automatically when the unit test finished.

Reply:
keep this function only for ut